### PR TITLE
Annotate models

### PR DESCRIPTION
**Pull request by gbp**
## Relevant issue(s)

Connects to #4902

## Why was this needed?

Release is coming lets annotate the models so they are up-to-date

## Implementation notes

- To ensure defaults values don't change I set `TIME_ZONE=London` in config/general.yml. This matches what is set in production for WDTK
- Cleared the existing annotations with: `bundle exec annotate -d`
- Finally ran: `bundle exec annotate`

### CODE REVIEW

**Comment by garethrees** [comment]
Hmm, I'm convinced I'd run this myself.

I've always used `bundle exec annotate`. I noticed you used `bundle5` – I ran this as "Rails 4" as that's still the production version for release 0.33. 

I also never use `RAILS_ENV=production` – should we be doing that?

**Comment by lizconlan** [comment]
> I've always used bundle exec annotate. I noticed you used bundle5 – I ran this as "Rails 4" as that's still the production version for release 0.33.

Good point - I got confused for a moment there. We don't want the Rails 5 changes in release 0.33

> I also never use RAILS_ENV=production – should we be doing that?

Shouldn't make a difference but isn't needed as the models a) should be the same either way and b) the model docs are mostly useful for develop so running as production isn't ideal

**Comment by gbp** [comment]
I have force pushed a new commit based on comments. While redoing this noticed there are some difference with our production env. see: #5246

File: app/models/incoming_message_error.rb [hunk 1]
```diff
@@ -10,7 +10,6 @@
 #  retry_at   :datetime
 #  backtrace  :text
 #
-# models/incoming_message_error.rb:
```
Comment (lizconlan) [inline app/models/incoming_message_error.rb]: This looks like it's been removed in error - it wasn't originally added by annotate: https://github.com/mysociety/alaveteli/commit/eb944c6c7aebdd066ba897745155fb233335f789

Comment (gbp) [reply app/models/incoming_message_error.rb]: Ugh, why would annotate remove this. Think I'll add a blank line to prevent this from happening again.

**Comment by gbp** [comment]
> Now wondering why running annotate on my machine doesn't pick up the changes to factory code *sigh*

Hmm, maybe this also effects @garethrees as he was convinced he had already run this?

**Pull request merged**